runtime/lua/vim/lsp.lua

```diff
@@ -594,6 +594,14 @@ function lsp.enable(name, enable)
     })
 end
 
+--- Checks if the given LSP config is enabled or not.
```

Comment:
```suggestion
--- Checks if the given LSP config is enabled.
---
--- Note: use |vim.lsp.get_clients()| to check active LSP clients (as opposed to configs) for a given buffer.
```

Reply:
Done. Just wondering: this phrasing sort of implies that you can enable/disable a LSP config per buffer. Is that possible?

Reply:
No, the phrasing is intended to avoid confusion by being explicit. Clients are enabled per-buffer, and it's likely that users will get that confused with configs, which are (roughly) global.